lib/ae_mdw/db/sync/transaction.ex

```diff
@@ -223,6 +226,61 @@ defmodule AeMdw.Db.Sync.Transaction do
     {mutations, {txi + length(mb_txs), mbi + 1}}
   end
 
+  defp tx_mutations(
+         :contract_create_tx,
+         tx,
+         _signed_tx,
+         txi,
+         tx_hash,
+         _block_index,
+         block_hash,
+         mb_events
+       ) do
+    contract_pk = :aect_create_tx.contract_pubkey(tx)
+    owner_pk = :aect_create_tx.owner_pubkey(tx)
+    events = Map.get(mb_events, tx_hash, [])
+
+    mutations = [
+      ContractEventsMutation.new(contract_pk, events, txi)
```

Comment:
Would leave this outside of `tx_mutations` for being specific for `:contract_create_tx` and `:contract_call_tx` (to help decreasing the signature and make it clear which kinds of txs depend on the events).

Reply:
Not sure what you meant by this? `code >>>>> words`

Reply:
The proposal is to shorten the number of params of `tx_mutations` (sorry old saying from C, function signature) and keep the handling of `ContractEventsMutation` where it was. With also the removal of `signed_tx`, in code terms it would become like this:
```
tx_mutations(
         :contract_create_tx,
         tx,
         txi,
         tx_hash,
         _block_index,
         block_hash
       )
```
Beyond making the function call simpler for other code readers, most importantly, it keeps it clear where the `mb_events` and `signed_tx` are used by being explicit (avoid searching in the different mutations how/when it's used).

PS: From preexisting code, following this same reasoning, all microblocks shouldn't be passed to a `transaction_mutations`. A better parameter would be `tx_events` as a tx doesn't look for events from other tx.